benchmark/perf-tests/100-rayjob/pytorch-mnist-rayjob.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: {{.Name}}
+  labels:
+    perf-test: rayjob-pytorch-mnist
+spec:
+  shutdownAfterJobFinishes: true
+  entrypoint: python ray_train_pytorch_mnist.py
+  runtimeEnvYAML: |
+    env_vars:
+      NUM_WORKERS: "2"
+      CPUS_PER_WORKER: "1"
+  rayClusterSpec:
+    rayVersion: '2.9.3'
+    headGroupSpec:
+      rayStartParams: {}
+      template:
+        spec:
+          containers:
+            - name: ray-head
+              image: {{.Image}}
+              ports:
+                - containerPort: 6379
+                  name: gcs-server
+                - containerPort: 8265
+                  name: dashboard
+                - containerPort: 10001
+                  name: client
+              env:
+              - name: "OMP_NUM_THREADS"
```

Comment:
Move to `runtimeEnvYAML` instead? Could you also add some comments in the YAML to explain why do we need to set this? Thanks!

Reply:
done